torchao/dtypes/__init__.py

```diff
@@ -1,7 +1,11 @@
 from .nf4tensor import NF4Tensor, to_nf4
 from .uint4 import UInt4Tensor
 from .aqt import AffineQuantizedTensor, to_aq
-from .float6_e3m2 import to_float6_e3m2, from_float6_e3m2
+
+try:
+    from .float6_e3m2 import to_float6_e3m2, from_float6_e3m2
+except RuntimeError:
+    pass
```

Comment:
nit: if these are not imported we can't include them in the `__all__` list in L16-17?